addon/bootstrap.js

```diff
@@ -40,6 +40,7 @@ const REASONS = {
   ADDON_UPGRADE: 7,    // The add-on is being upgraded.
   ADDON_DOWNGRADE: 8,  // The add-on is being downgraded.
 };
+
 for (const r in REASONS) { REASONS[REASONS[r]] = r; }
```

Comment:
See [this comment](https://github.com/Alexrs95/shield-study-rappor/pull/2#discussion_r135037005)